domains/robonamari.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "robonamari",
+           "email": "ROBONAMARI@GMAIL.COM",
+           "discord": "905561025829548113"
+        },
+    
+        "record": {
+            "TXT": "_discord.robonamari.is-a.dev"
```

Comment:
`TXT` records does nothing to your website, also for discord's TXT verification you need to make another domain `_discord.robonamari.is-a.dev` with TXT record (content) given by discord